docker/deployment/app-deployment.yaml

```diff
@@ -110,7 +110,12 @@ spec:
       initContainers:
       - name: initialize-volume-ownership
         image: busybox:1.33
-        command: ['sh', '-c', 'chown www-data:www-data /var/www/html/assets /var/lib/languageforge/lexicon/sendreceive']
+        command:
+          - 'sh'
+          - '-c'
+          - |-
+            mkdir /var/lib/languageforge/lexicon/sendreceive/state
+            chown www-data:www-data /var/www/html/assets /var/lib/languageforge/lexicon/sendreceive /var/lib/languageforge/lexicon/sendreceive/state
```

Comment:
This isn't going to run two commands. The newline character is *not* a separator when running `sh -c`. This is going to try to create the following directories:

- `/var/lib/languageforge/lexicon/sendreceive/state`
- `chown`
- `www-data:www-data`
- and so on

You need a semicolon in between these arguments.

Reply:
yeah, that makes perfect sense, I need to double-check my tests and see if I missed something or this special syntax works differently for some reason, thanks for pointing it out.

Reply:
I double-checked, this syntax works fine and treats them as separate commands already, nothing to change here.